File: app/src/at-nhatnguyen/java/asiantech/internship/summer/javacore/Exercise16.java
package asiantech.internship.summer.javacore;
import java.util.Scanner;
public class Exercise16 {

    private static int input(){
        Scanner scan= new Scanner(System.in);
        boolean check= false;
        int n=0;
        while(!check){
            System.out.print(" ");
            try{
                n = scan.nextInt();
                check= true;
            }catch(Exception e){
                System.out.println("Ban phai nhap so :");
                scan.nextLine();
            }
        }
        return (n);
    }
    private static int countElement(int[] a, int n, int i){
        int count= 0;
        for(int j=0 ; j<n ; j++){
            if(a[j]== i)
                count ++;
        }
        return (count);
    }
    public static void main(String[] args) {
        int n,i;
        System.out.println("Nhap n= ");
        n= input();
        int[] array= new int[n];
        for(i=0 ; i<n ; i++){
            System.out.println("Nhap phan tu thu " +(i+1)+" ");
            array[i]= input();
        }
        for(i=0 ; i<n ; i++){
            if(countElement(array, i, array[i])==0){
                System.out.println("Phan tu "+array[i]+ " xuat hien "+countElement(array, n,
                        array[i])+" lan");
            }
        }
    }
}

File: app/src/at-nhatnguyen/java/asiantech/internship/summer/kotlin/Exercise16.kt
package asiantech.internship.summer.kotlin

private fun countElement(a: IntArray, n: Int, i: Int): Int {
    var count = 0
    for (j in 0 until n) {
        if (a[j] == i) {
            count++
        }
    }
    return count
}

fun main() {
    val n: Int = input()
    var i = 0
    print("Nhap n= ")
    val array = IntArray(n)
    while (i < n) {
        print("Nhap phan tu thu " + (i + 1) + " ")
        array[i] = input()
        i++
    }
    i = 0
    while (i < n) {
        if (countElement(array, i, array[i]) == 0) {
            println(
                    "Phan tu " + array[i] + " xuat hien " + countElement(
                            array, n,
                            array[i]
                    ) + " lan"
            )
        }
        i++
    }
}

